roles/common/templates/docker-compose.yml.j2

```diff
@@ -212,6 +212,7 @@ services:
     volumes:
       - ./nuxeo/{{ nuxeo_node }}.conf:/docker-entrypoint-initnuxeo.d/nuxeo.conf:ro
       - ./nuxeo/init-nuxeo.sh:/docker-entrypoint-initnuxeo.d/init-nuxeo.sh:ro
+      - ./nuxeo/docker-entrypoint-initnuxeo.d:/docker-entrypoint-initnuxeo.d:rw
```

Comment:
Do we need the :rw or :ro is enough ?
Should we create the docker-entrypoint-initnuxeo.d directory ?

Reply:
The docker-entrypoint-initnuxeo.d directory will be created automatically, rw is because of line 214, the init-nuxeo.sh is stored in the container in that folder as well and ro might break the mapping of that file. It should be more homogenous, I will rework that part later on.